app/controllers/api/v1/questions_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::QuestionsController < Api::BaseController
+  def daily
+    render json: Question.active.first
```

Comment:
Just for posterity we have already discuss this in Guilded, we will need to also filter by daily questions

Reply:
Yes, this is the first commit. I am going to change it.